src/sage/schemes/elliptic_curves/kraus.py

```diff
@@ -674,13 +681,13 @@ def check_Kraus_local(c4, c6, P, assume_nonsingular=False):
 
     - ``P`` -- a prime ideal of the number field
 
-    - ``assume_nonsingular`` (boolean, default False) -- if True,
-      check for integrality and nosingularity.
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
+      check for integrality and nonsingularity.
 
     OUTPUT:
 
-    Tuple: either (True,E) if there is a Weierstrass model E integral
-    at P and with invariants c4, c6, or (False, None) if there is
+    Tuple: either ``(True, E)`` if there is a Weierstrass model E integral
+    at P and with invariants c4, c6, or ``(False, None)`` if there is
```

Comment:
```suggestion
    Tuple: either ``(True, E)`` if there is a Weierstrass model `E` integral
    at `P` and with invariants `c_4`, `c_6`, or ``(False, None)`` if there is
```

Reply:
Here i move these invariants to format as Latex, but maybe this is not what we want